src/plone/volto/profiles/default/actions.xml

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0"?>
+<object xmlns:i18n="http://xml.zope.org/namespaces/i18n"
+        meta_type="Plone Actions Tool"
+        name="portal_actions"
+>
+  <action-provider name="portal_workflow" />
+  <action-provider name="portal_types" />
+  <action-provider name="portal_actions" />
```

Comment:
@nileshgulia1 Not needed.

Reply:
done [2ff8128](https://github.com/plone/plone.volto/pull/64/commits/2ff81286ac357d80e7b4b0898ef2e9d284fb76f7)